benchmark/optimizer/bench_like_optimizer.cpp

```diff
@@ -0,0 +1,207 @@
+#include "benchmark_runner.hpp"
```

Comment:
Could you perhaps remove these benchmarks? They were useful for the PR but we want to limit the amount of benchmarks we have as they are all run on every commit.